Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -1145,13 +1145,16 @@ void AODProducerWorkflowDPL::fillMCTrackLabelsTable(MCTrackLabelCursorType& mcTr
                   break;
                 }
               }
+              if (isTOFFake) {
```

Comment:
@njacazio the line https://github.com/AliceO2Group/AliceO2/pull/13902/files#diff-6dc278732c5ed5f43d5567ad2c469ac11acf0e85339ae708bdf977ddd8fba392R1143 is simply wrong: it compares the original full MCCompLabel of TOF cluster with uint  label of TPC assigned for storage in the AOD! All these manipulations with `mToStore` container are extremely heavy and unnecessary: the MCCompLabel are much lighter. I'll fix this part, then let's continue from that point on.